msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -864,129 +864,129 @@ Delegated permissions allow the app to access data on behalf of the signed-in us
 |`tabs`| Array | |✔️| Configures custom tabs on the Microsoft 365 application ribbon. |
-|`tabs.id`| String | 64 characters | | Unique identifier for the tab within the app.|
-|`tabs.label`| String | 64 characters | | Text displayed for the tab.|
+|`tabs.id`| String | 64 characters | | Specifies the ID for the tab within the app.|
+|`tabs.label`| String | 64 characters | | Specifies the text displayed for the tab.|
 |`tabs.position`| Object | | | Configures the position of the custom tab relative to other tabs on the ribbon.|
-|`tabs.position.builtinTabId`| String | 64 characters | | The ID of the built-in tab. For more information, see [find the IDs of controls and control groups](/office/dev/add-ins/design/built-in-button-integration#find-the-ids-of-controls-and-control-groups).|
+|`tabs.position.builtinTabId`| String | 64 characters | | Specifies the ID of the built-in tab. For more information, see [find the IDs of controls and control groups](/office/dev/add-ins/design/built-in-button-integration#find-the-ids-of-controls-and-control-groups).|
 |`tabs.position.align`| String enum | | | Defines alignment of the custom tab relative to the specified built-in tab. Supported values: `after`, `before`|
 |`tabs.groups`| String |64 characters | | Defines the tab groups.|
-|`tabs.groups.id`| String |64 characters | | Unique identifier for the tab group within the app. It must be different from any built-in group ID in the Microsoft 365 application and any other custom group.|
-|`tabs.groups.label`| String | 64 characters | | Text displayed for the group. |
-|`tabs.groups.icons`| Array | | | Icons displayed for the group. |
-|`tabs.groups.icons.size`| Number | |✔️| Size of the icon in pixels, enumerated as `16`,`20`,`24`,`32`,`40`,`48`,`64`,`80`. Required image size: `16`, `32`, `80`. |
-|`tabs.groups.icons.url`| URL| | | URL of the icon.|
-|`tabs.groups.icons.file`| String | 2048 characters| | Relative path to the file that contains the icon. This property isn’t supported as a relative path. |
+|`tabs.groups.id`| String |64 characters | | Specifies the ID for the tab group within the app. It must be different from any built-in group ID in the Microsoft 365 application and any other custom group.|
+|`tabs.groups.label`| String | 64 characters | | Specifies the text displayed for the group. |
+|`tabs.groups.icons`| Array | | | Specifies the icons displayed for the group. |
+|`tabs.groups.icons.size`| Number | |✔️| Specifies the size of the icon in pixels, enumerated as `16`,`20`,`24`,`32`,`40`,`48`,`64`,`80`. Required image size: `16`, `32`, `80`. |
+|`tabs.groups.icons.url`| URL| | | Specifies the URL of the icon.|
+|`tabs.groups.icons.file`| String | 2048 characters| | Specifies the relative path to the file that contains the icon. This property isn’t supported as a relative path. |
```

Comment:
Update the `url` property to be "Specifies the absolute URL of the icon." 
Remove the `file` property entirely.